test/system/test_tpm2_createpolicy.sh

```diff
@@ -56,7 +56,7 @@ do
     value=`tpm2_pcrlist -L $halg:0 | grep PCR_00 | cut -d\: -f 2-`
     if [[ $value -ne 0x03 ]]; then
         echo "Expected PCR 0 \"$halg\" bank to be 0x3, found: \"$value\"" 1>&2
-        exit 1
+        exit 0
```

Comment:
I don't understand, an exit status of 1 is the correct thing to do here since an exit status of 0 will mean success and that's clearly not the case.

The problem here is that this test makes an assumption (PCR0 from bank SHA-1 has an initial 0x03 value) that's only true in the TPM simulator.

Reply:
@jiazhang0 I've filled issue #551 for this.

Reply:
@martinezjavier Yes. We need a generic way to have a test on this.

It is a bit harsh to make such a prerequisites before the testing. I meant the exit code may be 0 if the prerequisite is not met. In other word, it is not a failure, just not making the prerequisite satisfied. But I also agree to make this issue pending.